addons/web/static/src/core/datepicker/datepicker.js

```diff
@@ -146,12 +146,13 @@ export class DatePicker extends Component {
         if (typeof commandOrParams === "object") {
             const format = luxonFormatToMomentFormat(this.props.format || this.defaultFormat);
             const params = { ...commandOrParams, format };
+            params.locale = params.locale || commandOrParams.date.locale;
             for (const prop in params) {
                 if (params[prop] instanceof DateTime) {
                     const luxonDate = params[prop];
                     const momentDate = luxonDateToMomentDate(luxonDate);
                     if (momentDate.locale() !== luxonDate.locale) {
-                        momentDate.locale(params.locale);
+                        momentDate.locale(luxonDate.locale);
```

Comment:
is it true for all available locales ?

Reply:
After reading the code more carefully I think they actually may be useless: the locale assigned here is not the one given in the `params` object, however the test asserting that the `params.locale` is applied works so I assume that the locale of individual dates is ignored by bootstrap, so I think these 3 lines are unnecessary.

Reply:
lines removed @Arcasias